packages/protocol/contracts/stability/StableToken.sol

```diff
@@ -392,6 +393,10 @@ contract StableToken is
   /**
    * @notice Returns the exchange id in the registry of the corresponding fiat pair exchange.
    * @dev When this storage is uninitialized, it falls back to the default EXCHANGE_REGISTRY_ID.
+   * exchangeRegistryId was introduced after the initial release of cUSD's StableToken,
```

Comment:
not sure if we render the natspec anywhere but I'm not sure how it will pick up these multiline comments

Reply:
There are some other multiline comments that seem fine-- I can't really find anything online to confirm whether or not it adheres to natspec though
https://github.com/celo-org/celo-monorepo/blob/master/packages/protocol/contracts/stability/StableToken.sol#L535